quickstart.rst

```diff
@@ -4,7 +4,9 @@
 Quick Start
 ============
 
-A quick introduction to Flux and flux-core.
+A quick introduction to Flux and flux-core. For reference, the :ref:`Glossary`
```

Comment:
I think this is good for the first shot - what we will eventually do is define a term later and then put a link to it so the person can click and immediately be taken to the exact definition. We can also look into if it might be possible to do some kind of popup so they don't leave the page.